airbyte-json-validation/src/main/java/io/airbyte/validation/json/JsonSchemaValidator.java

```diff
@@ -57,7 +57,16 @@ public JsonSchemaValidator() {
   }
 
   public boolean test(JsonNode schemaJson, JsonNode objectJson) {
-    return validate(schemaJson, objectJson).isEmpty();
+    Set<ValidationMessage> validationMessages = validate(schemaJson, objectJson);
+
+    if (!validationMessages.isEmpty()) {
+      String str = String.format(
+          "JSON schema validation failed. \nerrors: %s",
+          Strings.join(validationMessages, ", "));
+      System.out.println(str);
```

Comment:
For logging, it's best to use a `Logger` object. Typically we declare a `LOGGER` at the top of the file and use something like `LOGGER.info(..)`. Here are examples: 

* [Defining the logger](https://github.com/airbytehq/airbyte/commit/38e84f1746b4c35b92dd646514cf67921badb7e1#diff-b5220bb6ab91f624515384c9c4a153c5116176cc79eb7f0835029bd326736883R35)
* [Using the logger](https://github.com/airbytehq/airbyte/commit/38e84f1746b4c35b92dd646514cf67921badb7e1#diff-b5220bb6ab91f624515384c9c4a153c5116176cc79eb7f0835029bd326736883R55)

Reply:
Fixed that